test/stub/keyring-bridge.js

```diff
@@ -0,0 +1,55 @@
+export const KNOWN_PUBLIC_KEY =
+  '02065bc80d3d12b3688e4ad5ab1e9eda6adf24aec2518bfc21b87c99d4c5077ab0';
```

Comment:
maybe we could think on adding the derived addresses from this pub key too, so we can easily access them for testing? (relates to my comment in the test below) :thinking: wdyt?

I see that these are some derived addresses, from the `metamask-controller` test:

```
const KNOWN_PUBLIC_KEY_ADDRESSES = [
  {
    address: '0x0e122670701207DB7c6d7ba9aE07868a4572dB3f',
    balance: null,
    index: 0,
  },
  {
    address: '0x2ae19DAd8b2569F7Bb4606D951Cc9495631e818E',
    balance: null,
    index: 1,
  },
  {
    address: '0x0051140bAaDC3E9AC92A4a90D18Bb6760c87e7ac',
    balance: null,
    index: 2,
  },
  {
    address: '0x9DBCF67CC721dBd8Df28D7A0CbA0fa9b0aFc6472',
    balance: null,
    index: 3,
  },
  {
    address: '0x828B2c51c5C1bB0c57fCD2C108857212c95903DE',
    balance: null,
    index: 4,
  },
];
``

Reply:
I moved the known addresses from the unit test file to `keyring-bridge.js` so that we can easily find them in one place: [67637d2](https://github.com/MetaMask/metamask-extension/pull/25824/commits/67637d24e4da2a0b28c75e1f58da53b56c8d6bf7)

Reply:
Hmm, seems like unit tests are failing after this change

Reply:
mm interesting! I guess it's also okay as it is now, we can access from the e2e tests to the public addresses :+1: